### DISCUSSION TITLE
Dose data entry at steady state

### DISCUSSION THREAD

**Discussion by VishnuEriyat** [LAPKB/Pmetrics#191] [Q&A] [answered]
I was trying to construct a model for oral rifampicin in adults with pulmonary tuberculosis. There are patients with observation at multiple visits (1 month, 2 month, 5 months etc). When I enter the dose at the in between visits say at 2 months should I put it as EVID 4 and should the ADDL be also set to the number of doses between the last and current visit.

**Comment by mnneely** [comment, marked as answer]
Including a bunch of unwitnessed doses between PK observation events will bog down your analysis. I would use the EVID = 4 flag as you said. So the first record will be a dose with time 0 and EVID = 1. If you have additional doses you want to include in that block, you can use ADDL, e.g. 5 additional dose at II interval prior to the PK sampling. Or add them manually with specific times and EVID = 1. Then enter your observations for that block with EVID = 0. This is all the normal and typical setup.

Now comes the part where you “jump” over all those unobserved doses. The next dose record would be again at time = 0 but now with an EVID = 4. This resets all compartments to 0, and generates a new “block” under the hood, which starts at time 0 like all other blocks. You probably will want to use ADDL to add additional doses in this new block, or specify them manually if you have the exact times. Follow with observations.

EVID = 4 is a reset event. It is only used as the first dose in a new block of doses/observations. It should only be used to begin the second block and after for a given subject. Doses within any block after the EVID = 4 dose are coded with EVID = 1 like any other dose, and the times should increase monotonically from zero.

I know that can be complex, but I hope this explanation clarifies.

**Reply by VishnuEriyat** [reply]
Thanks a lot for the prompt reply, that clarifies the point
